<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSide</class>
 <widget class="QWidget" name="ServerSide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="clientListVerticalLayout">
       <item>
        <widget class="QLabel" name="loginClientListLabel">
         <property name="text">
          <string>Login Client List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="clientTreeWidget">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Image</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="logoutClientListLabel">
         <property name="text">
          <string>Logout Client List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="clientQuitTreeWidget">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Image</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="logDataLabel">
         <property name="text">
          <string>Log Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="messageTreeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Port</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="clientListComboBox"/>
         </item>
         <item>
          <widget class="QLineEdit" name="sendLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="sendPushButton">
           <property name="text">
            <string>전송</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="messageLayout">
         <item>
          <widget class="QPushButton" name="clearPushButton">
           <property name="text">
            <string>로그 위젯 비우기</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savePushButton">
           <property name="text">
            <string>로그 저장하기</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitPushButton">
           <property name="text">
            <string>종료(Exit)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
